src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Calculator {
+    private static double x;
+    private static double y;
+    private static char operation;
+
+    public Calculator(double x, double y, char operation) {
+        Calculator.x = x;
+        Calculator.y = y;
+        Calculator.operation = operation;
+    }
+
+    public double calculate() {
```

Comment:
```suggestion
    public double calculate(double x, double y, char operation) {
```
why do we need to create a new Calculator each time we want to do a calculation?